### PR TITLE
* Rebuild embed & add social connect button

### PR DESCRIPTION
**Pull request by nwingt**

### CODE REVIEW

File: pages/in/embed/_id/index.vue [hunk 1]
```diff
@@ -111,6 +111,12 @@ export default {
         };
       })
       .catch(() => {});
+
+    data.platforms = await apiGetSocialListById(params.id)
```
Comment (williamchong007) [inline pages/in/embed/_id/index.vue]: probably should user `Promise.All()` for the 2 api, each with its own `catch`?

Comment (nwingt) [reply pages/in/embed/_id/index.vue]: Updated

File: pages/in/embed/_id/index.vue [hunk 2]
```diff
@@ -97,20 +90,23 @@ export default {
...
-          displayName,
-          avatar,
-          amount,
-        };
-      })
-      .catch(() => {});
+    const { id } = params;
+    return Promise.all([
+      apiGetUserMinById(id).catch(() => {}),
+      apiGetSocialListById(id).catch(() => {}),
```
Comment (williamchong007) [inline pages/in/embed/_id/index.vue]: 下面會 `.data()` ， catch 應該要 return object？
```javascript
() => ({})
```

Comment (williamchong007) [reply pages/in/embed/_id/index.vue]: 另外可能 not found id 應該 redirect 404

Comment (williamchong007) [reply pages/in/embed/_id/index.vue]: e.g. https://github.com/likecoin/likecoin-store/blob/master/pages/_id/index.vue#L224

Comment (nwingt) [reply pages/in/embed/_id/index.vue]: Error 應該俾上面獨立catch catch晒，所以唔洗再`Promise.all().catch()`

Comment (williamchong007) [reply pages/in/embed/_id/index.vue]: i mean 而家 catch 咗之後都係會 deference `res[1].data` 應該會搞到 throw error
不如係上面嘅 catch 用 `error()` send 佢去 404 page

**Pull request merged**